docs/architecture/adr-027-deterministic-protobuf-serialization.md

```diff
@@ -45,11 +46,18 @@ with the following additions:
    must be omitted
 4. `repeated` fields of scalar numeric types must use
    [packed encoding](https://developers.google.com/protocol-buffers/docs/encoding#packed)
-   by default.
-5. Variant encoding of integers must not be longer than needed.
+5. Variant encoding must not be longer than needed:
```

Comment:
Are we talking here about `variant`s or `varint`s?

Reply:
Ah, stupid autocorrect. That's supposed to be `varint`

Reply:
Fixed in 2c78cbe.